### ISSUE TITLE
DuckDB cannot read older version and vice versa

### ISSUE THREAD

**Issue by ybkamaleri** [helseprofil/orgdata#271]
Error message:

```
Error: rapi_startup: Failed to open database: IO Error: Trying to read a database file with version number 31, but we can only read version 33.
The database file was created with an older version of DuckDB.

The storage of DuckDB is not yet stable; newer versions of DuckDB cannot read old database files and vice versa.
The storage will be stabilized when version 1.0 releases.
```

**Comment by aeeeeeep** [comment]
How did you fix this bug?

**Comment by ybkamaleri** [comment]
It's a pragmatic solution and the fix is actually in #280 in this [file](https://github.com/helseprofil/orgdata/blob/6b6712915492d48c9552eccce08c73419787b29a/R/make-file.R#L99)
